Scan/ScanLib/source/XiaListModeDataMask.cpp

```diff
@@ -178,10 +223,13 @@ const {
         case R30981:
```

Comment:
This comment should be on line 216.

The throw has wrong message, it should be:
```c++
         throw invalid_argument(BadMaskErrorMessage
                                        ("GetEventEnergyMask"));
```